kojiclient.py

```diff
@@ -2,6 +2,29 @@
 import json
 from utils import getScriptDir
 
+tag2branch = {
```

Comment:
Can you define ``tag2branch`` as a function? This way you can compute the mapping generically. Otherwise new mapping will have to be added for each new distribution manually.